packages/devtools/lib/src/main.dart

```diff
@@ -72,10 +72,18 @@ class PerfToolFramework extends Framework {
     await addScreens();
     screensReady.complete();
 
+    final settingsScreen = getScreen('settings');
+    queryId('settings-button')
+      ..setAttribute('href', settingsScreen.ref)
+      ..onClick.listen((e) {
+        e.preventDefault();
+        navigateTo(settingsScreen.id);
+      });
+
     final CoreElement mainNav = CoreElement.from(queryId('main-nav'));
     mainNav.clear();
 
-    for (Screen screen in screens) {
+    for (Screen screen in screens.where((screen) => screen.id != 'settings')) {
```

Comment:
alternately maybe add a boolean to the Screen class that indicates that the screen should not be included in the main nav bar.

Reply:
ok, updated